.github/workflows/docker-publish.yml

```diff
@@ -5,13 +5,13 @@ on:
     # Publish `master` as Docker `latest` image.
     branches:
       - master
+      - fix-docker-versioning
```

Comment:
Wait, actually this should be removed again now that the "test job" finished successfully.